src/main/java/codeu/controller/ProfilePageServlet.java

```diff
@@ -0,0 +1,74 @@
+package codeu.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+import java.util.UUID;
+import java.time.Instant;
+
+/**
+ * Servlet class responsible for user registration.
+ */
+public class ProfilePageServlet extends HttpServlet {
+
+  /**
+   * Store class that gives access to Users.
+   */
+  private UserStore userStore;
+
+  /**
+   * Set up state for handling registration-related requests. This method is only called when
+   * running in a server, not when running in a test.
+   */
+  @Override
+  public void init() throws ServletException {
+    super.init();
+    setUserStore(UserStore.getInstance());
+  }
+
+  /**
+   * Sets the UserStore used by this servlet. This function provides a common setup method
+   * for use by the test framework or the servlet's init() function.
+   */
+  void setUserStore(UserStore userStore) {
+    this.userStore = userStore;
+  }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) 
+    throws IOException, ServletException { 
+    request.getRequestDispatcher("/WEB-INF/view/profilepage.jsp").forward(request, response);
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) 
```

Comment:
Let's remove this entire doPost() for now, then add back exactly what we need to actually do profile updates.